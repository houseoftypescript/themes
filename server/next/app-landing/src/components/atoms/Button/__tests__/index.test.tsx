import { render } from '@testing-library/react';
import { GradientButton } from '../index';

describe('GradientButton', () => {
  it('to match snapshot', () => {
    const { container } = render(<GradientButton scrolled />);
    expect(container).toMatchSnapshot();
  });
});
