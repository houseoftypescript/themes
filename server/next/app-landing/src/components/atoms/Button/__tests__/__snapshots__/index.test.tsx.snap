// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GradientButton to match snapshot 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1m0onwi-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</div>
`;
